contributivity_measures.py

```diff
@@ -39,22 +41,37 @@ def compute_independent_scores(
     for node in node_list:
         performance_scores.append(
             fl_training.compute_test_score_for_single_node(
-                node, epoch_count, single_partner_test_mode, global_x_test, global_y_test
+                node, epoch_count, single_partner_test_mode, x_test, y_test
             )
         )
+        the_scenario.fit_count += 1
```

Comment:
This seems strange. In the `compute_test_score_for_single_node` there will be multiple fits (1 for each epoch actually).